framework/CodeInterfaces/PHISICS/phisicsdata.py

```diff
@@ -0,0 +1,961 @@
+      for line in outfile:
+        line = self.removeSpaces(line)
+        stringIsFloatNumber = self.isFloatNumber(line.split(','))
+        if stringIsFloatNumber is True:
+          if (float(line.split(',')[0]) == float(matchedTimeSteps[timeStepIndex])):
+            line = re.sub(r'\n',r'',line)
+            for i in xrange (0,len(self.isotopeListMrtau)):
+              depList.append(line.split(',')[i+1])
+    return depList
+    
+  def findDecayHeat(self,line):
+    """
+      Determines whether the decay heat is printed.
+      @ In, line, string
+      @ Out, findDecayHeat, boolean, True if decay heat is printed, false otherwise
+      @ Out, DecayHeatUnits, string
+    """
+    DecayHeatUnits = None
+    isDecayHeatPrinted = line.split(' ')[-2]
+    if isDecayHeatPrinted == 'YES':
+      DecayHeatUnits =  line.split(' ')[-1]
+      return True, DecayHeatUnits
+    else:
+      return False, DecayHeatUnits
+  
+  def numberOfMediaUsed(self,mpi,numberOfMPI):
+    """
+      Finds the number of media used in a given INSTANT output.
+      @ In, mpi, integer, MPI number considered
+      @ In, numberOfMPI, integer, total number of MPI user-selected
+      @ Out, mediaUsed, integer, number of media treated in one INSTANT MPI output
+    """
+    count = 0
+    if self.phisicsRelap is True and mpi >= numberOfMPI - 1:
+      pass
+    else:
+      with open(os.path.join(self.workingDir, self.instantOutputFileMPI[mpi])) as outfile:
+        for line in outfile:
+          if re.search(r'Medium\s+\d+\s+used',line):
+            count = count + 1
+    return count
+
+  def getCPUtime(self,numberOfMPI):
+    """
+      Gets the PHISICS CPU time.
+      @ In, numberOfMPI, interger, number of MPI user-selected
+      @ Out, cpuTime, list, cpu time (string) in a list
+    """
+    cpuTimes = []
+    for mpi in xrange (0,numberOfMPI):
+      with open(os.path.join(self.workingDir,self.instantOutputFileMPI[mpi])) as outfile:
+        for line in outfile:
+          if re.search(r'CPU\s+time\s+\(min\)',line) and self.phisicsRelap is False:
+            cpuTimes.append(line.strip().split(' ')[-1])
+          if self.phisicsRelap is True:
+            return 'na'
+    cpuTime = self.summedListValues(cpuTimes)
+    return [cpuTime]
+          
+  def locateMaterialInFile(self,numberOfMPI):
+    """
+      Finds the material names in a given INSTANT output file.
+      @ In, numberOfMPI, integer, number of MPI user-selected
+      @ Out, materialsDict, dictionary, dictionary listing the media treated in a given mpi output.
+              format: {MPI-0:{1:fuel1_1, 2:fuel1_5, 3:{fuel1_7}}, MPI-2:{1:fuel1_2, 2:fuel1_3, 3:fuel1_4, 4:fuel1_6}}
+    """
+    materialsDict = {}
+    for mpi in xrange(0,numberOfMPI):
+      count = 0
+      materialsDict['MPI-'+str(mpi)] = {}
+      mediaUsed = self.numberOfMediaUsed(mpi,numberOfMPI)
+      if self.phisicsRelap is True and mpi >= numberOfMPI - 1 : pass
```

Comment:
why pass???
it is usefulness

Reply:
I need to ignore this (as explained above), because the mpi naming in PHISICS/RELAP is different as the naming in PHISICS standalone.